xain_fl/serve.py

```diff
@@ -24,11 +22,6 @@ def serve(coordinator: Coordinator, store: AbstractStore, server_config: ServerC
     This methods just creates the :class:`xain_fl.coordinator.coordinator.Coordinator`,
     sets up all threading events and threads and configures and starts the gRPC service.
```

Comment:
i know this docstring was already there but since you're touching it now: i think we can rewrite this better without so many "and"s!